pkg/action/printer.go

```diff
@@ -20,21 +20,25 @@ import (
 	"fmt"
 	"io"
 	"strings"
+	"time"
 
+	"helm.sh/helm/pkg/chartutil"
 	"helm.sh/helm/pkg/release"
 )
 
-// PrintRelease prints info about a release
-func PrintRelease(out io.Writer, rel *release.Release) {
+// PrintRelease prints info about a release. If fullInfo is true, it will print
+// the user supplied values and the computed values used to render the chart
+func PrintRelease(out io.Writer, rel *release.Release, fullInfo bool) error {
```

Comment:
Breaking out the fullInfo logic into its own function would allow you to refactor out the boolean flag

Reply:
The reason it is in here is for ordering purposes. I tried to think of a way that would preserve the current output formatting order with a separate function, but if we want to maintain the input the same, it needs to be in the middle. Basically, I could have this function take a function parameter that can insert things (like `func PrintRelease(out io.Writer, rel *release.Release, extras ExtraFunc...)`). But as this is specifically for printing a whole release, I feel like it makes sense here. Any ideas how I could make it cleaner?

Reply:
We can also break the current output ordering, which would probably be fine given how this is already breaking other things. That would also solve this issue